simplified-app-shared/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- Action/Navigation Strings -->
+  <string name="nav_confirm">Confirm</string>
+  <string name="nav_done">Done</string>
+  <string name="nav_next">Next</string>
 
   <!-- EULA strings -->
   <string name="eula_disagree">Disagree</string>
   <string name="eula_agree">Agree</string>
+  <string name="eula_checkbox">I agree to the terms of the End User License Agreement.</string>
+
+  <!-- Age Verification strings -->
+  <string name="age_verification_title">Age Verification</string>
+  <string name="age_verification_question">You must be 13 years of age or older to download a book from the collection.\nHow old are you?</string>
+  <string name="age_verification_changed">If you are under 13, all content downloaded to My Books will be removed.\nHow old are you?</string>
+  <string name="age_verification_13_older">13 or Older</string>
+  <string name="age_verification_13_younger">Under 13</string>
+  <string name="age_verification_13_checkbox">I am 13 years of age or older.</string>
+  <string name="age_verification_too_young">You are not old enough to sign up for a library card.r</string>
```

Comment:
is the "card.r" a typo?  and... gosh we're harsh.

Reply:
Typo!